java/src/main/java/org/vowpalwabbit/spark/VowpalWabbitNative.java

```diff
@@ -52,6 +64,18 @@ public class VowpalWabbitNative implements Closeable {
      */
     public native Object predict(VowpalWabbitExample[] examples);
 
+    private native Object predictFromString(String examplesString);
+
+    /**
+     * Invoke multi-line prediction. By default will interpret input as text format, pass `--dsjson` or `--json` to initialize to use DSJSON or JSON respectively.
+     *
+     * @param examplesString String representation of examples to learn from.
+     * @return the prediction.
+     */
+    public Object predict(String examplesString) {

```

Comment:
remove it. no need to duplicate.

Reply:
I added these so it just behaves as an overload rather than needing to call a different method. You don't think that's necessary?

Reply:
I think the learnFromString is obvious enough. duplicate methods usually lead to confusion.

Reply:
Sounds good